libs/build_jpeg.sh

```diff
@@ -40,11 +42,14 @@ sourceDir=$PWD
 [[ -d build ]] && rm -rf build
 mkdir -p build && cd build
 
+[[ $enable_shared =~ [yYtT] ]] && shared_flags="" || shared_flags="-DBUILD_STATIC=ON"
+
 cmake $sourceDir \
   -DCMAKE_INSTALL_PREFIX=$prefix \
   -DBUILD_TESTS=ON \
   -DBUILD_EXECUTABLES=ON \
-  -DCMAKE_BUILD_TYPE=RELEASE
+  -DCMAKE_BUILD_TYPE=RELEASE \
+  ${shared_flags}
```

Comment:
This is fine. 
Can you see `bugfix/jpeg` and please add the `checks_flag` logic from that branch.  It will save us another PR.

Reply:
Should be done now, please check if I missed something.